src/main/scala/io/iohk/ethereum/vm/EvmConfig.scala

```diff
@@ -105,6 +108,11 @@ object EvmConfig {
     opCodeList = AghartaOpCodes
   )
 
+  val PhoenixConfigBuilder: EvmConfigBuilder = config => PhoenixConfigBuilder(config).copy(
```

Comment:
Here it shoud be:
` val PhoenixConfigBuilder: EvmConfigBuilder = config => AghartaConfigBuilder(config).copy(`

Reply:
Yes, my bad